src/Serializer/AbstractSerializer.php

```diff
@@ -241,7 +241,14 @@ protected function serializeValue($value)
         }
 
         if (\is_object($value)) {
-            return 'Object ' . \get_class($value);
+            $objectId = null;
+            if (isset($value->id)) {
+                $objectId = $value->id;
+            } elseif (\is_callable([$value, 'getId']) && method_exists($value, 'getId')) {
+                $objectId = $value->getId();
+            }
+
+            return 'Object ' . \get_class($value) . ((null !== $objectId) ? '(#' . $objectId . ')' : '');
```

Comment:
```suggestion
            return 'Object ' . \get_class($value) . (null !== $objectId) ? '(#' . $objectId . ')' : '';
```

Reply:
I think the extra parentheses makes it more clear.